src/content/_index.en.md

```diff
@@ -24,6 +24,15 @@ include
 Like Open vSwitch, OVN is written in platform-independent C. OVN runs entirely
 in userspace and therefore requires no kernel modules to be installed.
 
+#### IRC meetings
+
+We have weekly IRC meetings you can participate in:
+
+[OVN weekly](http://eavesdrop.openstack.org/meetings/ovn_test/)
```

Comment:
I do not yet know the link we will use for eavesdrop.openstack.org; so let's hold off checking this in until the dust settles.